modular/gater/query_util.go

```diff
@@ -112,6 +114,23 @@ func (g *GateModular) checkSPAndBucketStatus(ctx context.Context, bucketName str
 	return nil
 }
 
+// checkSPAndBucketStatus check sp and bucket is in right status
+func (g *GateModular) getBucketTotalSize(ctx context.Context, bucketID uint64) (uint64, error) {
```

Comment:
the bucket size is allows changing.  ensure it has been called after migration start